src/smc-webapp/smc-react.coffee

```diff
@@ -557,8 +557,9 @@ react_component = (x) ->
 MODE = 'default'  # one of 'default', 'count', 'verbose', 'time'
 #MODE = 'verbose'
 #MODE = 'count'
-if not smc?
+if not window.smc?
```

Comment:
This will break importing everywhere that window isn't defined (so surely in the local-hub project, which imports this code for backend jupyter).  Either don't do this (for no reason?) or do `window?.smc?`.